lib/cfme/cloud_services/inventory_upload.rb

```diff
@@ -0,0 +1,56 @@
+module Cfme
+  module CloudServices
+    class InventoryUpload
+      def self.upload(targets)
+        targets = targets_from_queue(targets)
+
+        ManifestFetcher.fetch do |manifest|
+          DataCollector.collect(manifest, targets) do |payload|
+            DataPackager.package(payload) do |payload_path|
+              DataUploader.upload(payload_path)
+            end
+          end
+        end
+      end
+
+      def self.upload_queue(userid, targets)
+        task_opts = {
+          :action => "Collect and upload inventory to cloud.redhat.com",
```

Comment:
Minor, but I was thinking we should extract the "cloud.redhat.com" bit to a configuration setting...This would allow testing with a non-production cluster.  (not for this PR)

Reply:
Yeah once we have the Setting that @lpichler is adding I can use that